spec/formatting.md

```diff
@@ -349,17 +352,26 @@ The resolution of _markup_ MUST always succeed.
 
 ### Fallback Resolution
 
-A **_<dfn>fallback value</dfn>_** is the _resolved value_ for an _expression_ that fails to resolve.
+A **_<dfn>fallback value</dfn>_** is the _resolved value_ for
+an _expression_ or _variable_ that fails to resolve.
```

Comment:
Although this is pretty clear, it could be read as the _resolved value_ being the thing that fails to resolve.

```suggestion
an _expression_ or _variable_ when that _expression_ or _variable_ fails to resolve.
```